### ISSUE TITLE
Static API request with rate limits fail with no implicit conversion of String to Integer

### ISSUE THREAD

**Issue by hebron-george** [mikamai/ruby-lol#67]
``` ruby

client = Lol::Client.new '<my_api_key>', 
  region: 'na',
  rate_limit_requests: 10, 
  rate_limit_seconds: 3600

all_champions = client.static.champions.get 

# => project_path/vendor/ruby/2.4.0/gems/glutton_ratelimit-0.2.0/lib/glutton_ratelimit.rb:26: warning: constant ::Fixnum is deprecated
# => TypeError: no implicit conversion of String into Integer

```

I stepped through `static_request.rb` and the problem seems to be coming from https://github.com/mikamai/ruby-lol/blob/master/lib/lol/request.rb#L131-L133

The `times` method from `@rates_limiter` is making `perform_rate_limited_request` return `1` when `Lol::Client` is initialized with rate limit info.

I tried the exact same request except without the rate limit info:

``` ruby
client = Lol::Client.new '<my_api_key>', 
  region: 'na'
```

And that worked without any problems.

**Comment by hebron-george** [comment]
Opened PR #71

**Comment by baseballlover723** [comment]
This also affects the summoner endpoint as well (https://github.com/mikamai/ruby-lol/blob/master/lib/lol/summoner_request.rb#L23), that errors with `An hash is required as parameter`. This is due to `perform_rate_limited_request` returning one because of the rate_limiter. PR #71 should fix this as well. I'm guessing that this affects more endpoints as well.

**Comment by oeN** [comment]
solved #73
